influxdb2_client/src/models/onboarding.rs

```diff
@@ -0,0 +1,80 @@
+//! # Onboarding
+//!
+//! Initial setup of InfluxDB instance
+
+use serde::{Deserialize, Serialize};
+
+/// Check if database has default user, org, bucket created, returns true if
+/// not.
+#[derive(Clone, Copy, Debug, PartialEq, Default, Serialize, Deserialize)]
+pub struct IsOnboarding {
+    /// True if onboarding has already been completed otherwise false
+    #[serde(default)]
+    pub allowed: Option<bool>,
```

Comment:
Actually I meant removing the `Option` here and using the default of `false`, so:

```suggestion
    pub allowed: bool,
```

and then we should be able to remove the `unwrap` in influxdb2_client/src/api/setup.rs right?

Reply:
Ah right! I got it now :) Fixed!